lib/libutils/ext/include/util.h

```diff
@@ -30,6 +30,13 @@
 #include <compiler.h>
 #include <stdint.h>
 
+#define SIZE_4K	UINTPTR_C(0x1000)
+#define SIZE_1M	UINTPTR_C(0x100000)
+#define SIZE_2M	UINTPTR_C(0x200000)
+#define SIZE_4M	UINTPTR_C(0x400000)
+#define SIZE_8M	UINTPTR_C(0x800000)
+#define SIZE_2G	UINTPTR_C(0x80000000)
```

Comment:
Move to a separate commit.

Reply:
Looks ok to me. But is it the right format for asm/C to get the (unsigned long int) sized value ?

Reply:
ASM seems not recognize UL or L...etc. I cannot figure out what's the difference between UL and LU. But UINTPTR_C distinguishes ASM and C, so I used it.

Reply:
I think "LU" is a typo that slipped trough because the compiler seems to accept it.

Reply:
Actually, `UL` and `LU` are equivalent and are defined by the C99 standard in section 6.4.4.1:
```
integer-suffix:
    unsigned-suffix long-suffix opt
    unsigned-suffix long-long-suffix
    long-suffix unsigned-suffix opt
    long-long-suffix unsigned-suffix opt
unsigned-suffix: one of
    u U
long-suffix: one of
    l L
long-long-suffix: one of
    ll LL
```